docs/DECISIONS.md

```
@@ -373,6 +373,52 @@ Fetch Threads replies at build time via the Graph API using the denim library. C
 
 ---
 
+## ADR-014: Sveltia CMS Image Handling — Survey & Status Quo
+
+**Date**: 2026-04-27
+**Status**: Accepted
+
+### Context
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)